Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -923,4 +923,6 @@ It can efficiently compress both junk data and encrypted Wii data.
     <string name="incompatible_figure_selected">Incompatible Figure Selected</string>
     <string name="select_compatible_figure">Please select a compatible figure file</string>
 
+    <string name="emulate_wii_speak">Wii Speak</string>
+    <string name="disconnect_wii_speak">Mute Wii Speak</string>
```

Comment:
"Mute" doesn't match what the setting is called in the code or in DolphinQt. Is this something that was overlooked, or is it intentional?

Reply:
It is intended since it is presented differently on PC and Android. On PC it has its own dedicated window (similar to the emulated Infinity base):
![image](https://github.com/dolphin-emu/dolphin/assets/7890055/a72f3d1b-b31d-4413-b056-fd10c9ef7686)

Whereas on Android, it's only a toggle settings below the one enabling the Wii Speak. So to make it clearer for the end-user I decided to rename it (as the effect and purpose of disconnecting it is not to record audio). I feel that having a "Wii Speak" toggle, plus a "Disconnect Wii Speak" toggle right under it sounds strange from a UI point of view.

I'm open to better ways of presenting this option to the user.

Reply:
What does the setting do from the perspective of the emulated software? Does it disconnect the Wii Speak USB device, or does it just make the audio be silence? If it's the latter, I think both the Android UI and the PC UI are okay, but the internal name for the setting would make more sense as "mute" than "connect". "Connect" makes sense in the PC UI only because it's in the context of whether you want to connect a host microphone.